Bottom: 83f1bde3bd461116ddce5071eb66282bda7fbe98
Top:    c4a1105661ee26e1c1716b7f829d333206a1fde9
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2014-07-04 03:48:08 +0100

Refresh of try-to-avoid-warnings-from

---

diff --git a/src/parse_y.y b/src/parse_y.y
index 29db9df..6246705 100644
--- a/src/parse_y.y
+++ b/src/parse_y.y
@@ -131,7 +131,6 @@ static Coord new_units (PLMeasure m);
 
 parse
 		: parsepcb
-		| parsedata
 		| parsefont
 		| error { YYABORT; }
 		;
@@ -165,7 +164,7 @@ parsepcb
 
 				if (!yyPCB)
 				{
-					Message(_("illegal fileformat\n"));
+					Message(_("illegal fileformat 1\n"));
 					YYABORT;
 				}
 				for (i = 0; i < MAX_LAYER + EXTRA_LAYERS; i++)
@@ -240,25 +239,6 @@ parsepcb
 		  }
 		;
 
-parsedata
-		:	{
-					/* reset flags for 'used layers';
-					 * init font and data pointers
-					 */
-				int	i;
-
-				if (!yyData || !yyFont)
-				{
-					Message(_("illegal fileformat\n"));
-					YYABORT;
-				}
-				for (i = 0; i < MAX_LAYER + EXTRA_LAYERS; i++)
-					LayerFlag[i] = false;
-				yyData->LayerN = 0;
-			}
-		 pcbdata
-		;
-
 pcbfont
 		: parsefont
 		|
@@ -272,7 +252,7 @@ parsefont
 
 				if (!yyFont)
 				{
-					Message(_("illegal fileformat\n"));
+					Message(_("illegal fileformat 3\n"));
 					YYABORT;
 				}
 				yyFont->Valid = false;
